backend/src/database/entities/availability.entity.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  Column,
+  Entity,
+  PrimaryGeneratedColumn,
+  ManyToOne,
+  JoinColumn,
+} from 'typeorm';
+import { AvailabilityType } from '../../common/enums/availability-type.enum';
+import { PersonnelEntity } from './personnel.entity';
+
+@Entity('function')
+export class AvailabilityEntity {
+  @PrimaryGeneratedColumn('increment')
```

Comment:
number ids instead of uuid? (I'm fine with it just checking if this is the result of setting 'increment')

Reply:
Yep! I figured that there's no need to mask the ids for these, so I left UUID on personnel and ids on everything else